index.html

```diff
@@ -753,23 +756,29 @@ <h2>Queue the PerformanceObserver task</h2>
                 {{PerformanceObserverEntryList}}, with its <a>entry list</a> set
                 to <var>entries</var>.
                 </li>
-                <li>Let <var>droppedEntriesCount</var> be 0.</li>
-                <li>For each <a>PerformanceObserverInit</a> <var>item</var> in
-                <var>registeredObserver</var>'s <a>options list</a>:
+                <li>Let <var>droppedEntriesCount</var> be null.</li>
```

Comment:
Why "null"?

Reply:
Fixed! Null just means that we won't send that parameter.